### ISSUE TITLE
[Request] [urgent] Add links to image source

### ISSUE THREAD

**Issue by networxnet** [mapodev/impressum-manager-wordpress#10]
We missing the ability to add links to image source. This is required for many sources. Is it possible to add quickly?

**Comment by mapodev** [comment]
Not quickly. I can take a look at the weekend.

**Comment by networxnet** [comment]
Thanks for the quick reply. Do you have an idea if there is any workaround we could use so far? We have complaints with one important image source which insists to be linked immediately :(

**Comment by mapodev** [comment]
What do you mean by image source? Do you want to link an image to an URL?

**Comment by networxnet** [comment]
Impressum Manager generates a "Quellenangaben für die verwendeten Bilder und Grafiken" list and we want to set links for single sources. Maybe it is possible to edit the list manually? Or if i add additional fields with Advanced Custom Fields plugin [](http://www.advancedcustomfields.com/)

**Comment by mapodev** [comment]
Can you try to wrap a ahref tag around it?

Wir können auch deutsch schreiben.

**Comment by networxnet** [comment]
Ist schon Gewohnheit bei Github ;)

Das scheint nicht zu funktionieren. Habe Folgendes in das Urheber-Feld am Bild eingetragen: <a href="https://hi-flyfoto.de">Hi-FlyFoto</a>

**Comment by mapodev** [comment]
Das ist ja auch richtig. Wenn es anklickbar ist, dann funktioniert es doch anscheinend?

**Comment by mapodev** [comment]
Wenn nicht, dann wüsste ich erstmal kein Weg herum als außer den Link neben dem [impressum_manager] Tag.

**Comment by networxnet** [comment]
Nein, ist nicht anklickbar. Sorry hätte ich dazu schreiben sollen. Der Link wird weggeschluckt.
Daher meine Idee, ein zusätzliches Feld zu erzeugen und zu befüllen, dass für die Liste der Bildquellen zusätzlich angezogen wird.
Vor dem ersten Eintrag unserer Liste fehlt im Übrigen auch immer der Zeilenumbruch.

[www.levmussleben.eu/impressum](url)

**Comment by mapodev** [comment]
Ich saß an dem Projekt seit mehr als einem Jahr nicht mehr. Ich müsste mich in den Code reinlesen, das könnte ich erst am Wochenende.

**Comment by networxnet** [comment]
Ok, ich schaue ob mir auch noch was einfällt. Danke erstmal :)
